### ISSUE TITLE
Gracefully handle running ls with no data-bucket configured

### ISSUE THREAD

**Issue by punkstar** [Space48/magedbm2#19]

**Comment by tgerulaitis** [comment]
Should it also handle running with only `data-bucket`?

**Comment by vlad1992** [comment]
@tgerulaitis @punkstar can we have clarification on what is required here?

**Comment by tgerulaitis** [comment]
@vlad1992 IIRC, magedbm2 used to crash out if you didn't have `data-bucket` defined in the config, even though that bucket is only ever used if you require anonymised data exports. I believe the point of this issue is to allow running commands that do not require a data bucket, without the data bucket config defined.

On the flipside, if regular `bucket` setting is not defined, you should be able to run commands that only require a data bucket.

TLDR: if a setting is not required for a command, magedbm2 should not prevent you from running it without that setting defined.

**Comment by vlad1992** [comment]
- ls command should run whichever bucket is specified
- put command should run with or without either of the buckets but not without both depending on where the db goes
- get command should run without either of the buckets but not without both depending on where the db comes from
